### PR TITLE
fix: issue with tibble

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: 
  
## Why was it changed?
  
# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [ ] Package version bumped
- [ ] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: R/load_files.R [hunk 1]
```diff
@@ -811,8 +819,9 @@ read_EnVision_xlsx <- function(results_file, results_sheet) {
   # Find rows with data and drop empty columns
   colsRange <-
     grep("Plate information", unlist(df[, 1]))[1] + 10
+  colsToKeep <- which(colSums(is.na(df[seq_len(colsRange), ])) != colsRange)
   df <-
-    df[, colSums(is.na(df[seq_len(colsRange), ])) != colsRange]
+    df[, colsToKeep, with = FALSE]
```
Comment (j-smola) [inline R/load_files.R:824]: :+1

File: R/load_files.R [hunk 2]
```diff
@@ -142,6 +142,7 @@ read_in_manifest_file <- function(manifest_file, available_formats) {
           e
         ))
       })
+      data.table::setDT(df)
     } else if (manifest_ext %in% c("text/tsv",
```
Comment (j-smola) [inline R/load_files.R:146]: In this branch `data.table::setDT(df)` is also needed, as `utils::read.table` returns data.frame :(
There is also no test for that (scenario: manifest is tsv file)

File: R/load_files.R [hunk 3]
```diff
@@ -506,6 +507,7 @@ read_in_template_sheet_xlsx <- function(template_file, template_sheets, idx, pla
                    template_file[[idx]],
                    e))
     })
+    data.table::setDT(df)
```
Comment (j-smola) [inline R/load_files.R:510]: @bczech This one I do not understand.
Why it is needed here as in 496 line
```
df <-  df <- data.table::as.data.table(
        readxl::read_excel(
```
Maybe it should be?
```
 df <- data.table::serDT(
        readxl::read_excel(
````

File: R/load_files.R [hunk 4]
```diff
@@ -794,6 +800,7 @@ read_EnVision_xlsx <- function(results_file, results_sheet) {
           col_names = paste0("x", seq_len(48)),
           range = "A1:AV32"
```
Comment (j-smola) [inline R/load_files.R:801]: range = "A1:AV32"
 👀 Magic numbers?:

Comment (bczech) [reply R/load_files.R:801]: 552L is the answer :D

File: R/load_files.R (hunk 3 above)
Comment (bczech) [reply R/load_files.R:510]: good catch

Comment (j-smola) [reply R/load_files.R]: ✔️

File: R/load_files.R (hunk 2 above)
Comment (bczech) [reply R/load_files.R]: Done. Replaced `read.table` and `read.csc` by `data.table::fread`

File: R/load_files.R (hunk 4 above)
Comment (j-smola) [reply R/load_files.R]: ✔️

File: R/load_files.R (hunk 2 above)
Comment (j-smola) [reply R/load_files.R]: ✔️

**Review by j-smola — COMMENTED** [review]
Could you maybe ...
1. refactor also `parse_D300_metadata_file` as it returns tibble (expect_identical -> expect_equal in test)
2. change `read.table` -> `data.table::fread` in read_ref_data 
3. add to .gitignore : `*.DS_Store` :D

**Review by j-smola — APPROVED** [review]

**Pull request merged**